### PR TITLE
#164369285 create custom navbar

### PR DESCRIPTION
**Pull request by bobbylemm**
#### What does this PR do?
Create custom navbar component for the application
#### How should this be manually tested?
Clone the repo, then run `npm install` and `npm run start-dev`
Then on your browser visit `http://localhost:3000/`
#### What are the relevant pivotal tracker stories?
[#164369285](https://www.pivotaltracker.com/story/show/164369285)
#### Screenshots (if appropriate)
Screenshot 2019-03-12 at 12 32 49 PM
Screenshot 2019-03-12 at 11 02 08 AM

### CODE REVIEW

File: src/components/common/input/Input.jsx [hunk 1]
```diff
@@ -1,19 +1,21 @@
...
   width: ${props => props.inputWidth || '94%'};
   box-sizing: border-box;
   @media (max-width: 76.8rem) {
+    width: 100%;
+    padding: 0.8rem;
     flex-direction: column;
-    height: 3rem;
-    font-size: 1.5rem
+    height: 4.3rem;
+    font-size: 1.5rem;
```
Comment (omenkish) [inline src/components/common/input/Input.jsx:18]: Thank you for adding this.

File: src/components/common/input/Input.jsx [hunk 2]
```diff
@@ -1,19 +1,21 @@
...
   border-radius: 0.2rem;
   width: ${props => props.inputWidth || '94%'};
   box-sizing: border-box;
   @media (max-width: 76.8rem) {
+    width: 100%;
+    padding: 0.8rem;
     flex-direction: column;
-    height: 3rem;
-    font-size: 1.5rem
+    height: 4.3rem;
```
Comment (omenkish) [inline src/components/common/input/Input.jsx:17]: The initial plan was to reduce the height of the input on smaller screens.

**Review by omenkish — DISMISSED** [review]

**Review by divinediscipline — DISMISSED** [review]
Looks good at the moment

**Review by ozone4real — DISMISSED** [review]
LGTM

**Review by primuse — DISMISSED** [review]

**Review by biolabalo — DISMISSED** [review]

**Review by omenkish — DISMISSED** [review]

**Review by primuse — DISMISSED** [review]

**Review by biolabalo — DISMISSED** [review]

**Review by primuse — APPROVED** [review]

**Review by ozone4real — APPROVED** [review]
LGTM

**Review by EfeAgare — APPROVED** [review]

**Pull request merged**